dali/test/python/test_operator_decoders_image.py

```diff
@@ -90,26 +93,159 @@ def log(img_type, size, device, threads):
                     run_decode(data_path, batch_size, device, threads)
                     yield log, img_type, batch_size, device, threads
 
+# ToDo - check padding behavior
```

Comment:
```suggestion
# TODO(januszl): check padding behavior
```

Reply:
I don't know how useful if leaving the author next to TODO, Done.